docker-app/qfieldcloud/core/utils2/storage.py

```diff
@@ -344,6 +356,12 @@ def delete_file_version(
             versions_to_delete.append(file_version)
 
     for file_version in versions_to_delete:
+
+        if (
+            not re.match(r"^projects/[\w-]+/.+$", file_version._data.key)
+            or not file_version.version_id
```

Comment:
```suggestion
            or not file_version.id
```

Reply:
Ok committed, I saw that `version_id` in the stubs though (but nothing in the docs)
![image](https://user-images.githubusercontent.com/1894106/214600495-6e765322-5ddb-4eb2-b3d2-4fde939457b3.png)